core/src/main/scala/neotypes/DeferredQuery.scala

```diff
@@ -8,9 +8,6 @@ final case class DeferredQuery[T](query: String, params: Map[String, Any] = Map.
   def list[F[_]](session: Session[F])(implicit rm: ResultMapper[T]): F[List[T]] =
     session.transact(tx => list(tx))
 
-  def seq[F[_]](session: Session[F])(implicit rm: ResultMapper[T]): F[Seq[T]] =
```

Comment:
I thought of `seq` as a "more performant" alternative to list as it reuses underlying collection. But now I think that we're still doing lots of `map`s so there's overhead in any case.

Do you have any thought on it?

Reply:
Since I am using an iterator and the resulting list is built backwards and then reversed I think it will have good performance.
Anyways, if the result is too big for it to be considered bad, a **Stream** would always be the best alternative.

Finally, I do not really like returning a `Seq`.
Is a type which you can use safely, I prefer having a more concrete type like `List` or `Set`.
For a more detailed discussion about that, see [this](https://gist.github.com/djspiewak/2ae2570c8856037a7738#generic-interfaces).

Reply:
Sounds good